src/components/wizards/create-vm/StorageTable.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { TablePfProvider, selectionHeaderCellFormatter, selectionCellFormatter, Table } from 'patternfly-react';
+import { Header, Body} from 'reactabular-table';
+
+export class StorageTable extends React.Component {
+
+    state = {
```

Comment:
You can use this form in `CreateVmWizard` component too.

Let's be consistent with how we initialize component's state, using ES6 class fields sounds good to me.

Reply:
Ok, class fields it is.